cookbooks/aws-parallelcluster-entrypoints/recipes/update.rb

```diff
@@ -22,3 +22,18 @@
 include_recipe 'aws-parallelcluster-environment::directory_service'
 include_recipe 'aws-parallelcluster-slurm::update' if node['cluster']['scheduler'] == 'slurm'
 include_recipe 'aws-parallelcluster-scheduler-plugin::update' if node['cluster']['scheduler'] == 'plugin'
+
+# Update node package - useful for development purposes only
+if !node['cluster']['custom_node_package'].nil? && !node['cluster']['custom_node_package'].empty?
+
+  service "supervisord" do
+    action :stop
+  end
+
+  include_recipe 'aws-parallelcluster-shared::parallelcluster_custom_node'
```

Comment:
Why in is defined in the shared cookbook ?
Do we need it in other cookbook ? And if so, why ?

Reply:
Ok.. I saw you moved it in a following commit.